homu/server.py

```diff
@@ -29,8 +29,16 @@ def index():
 
 @get('/queue/<repo_label>')
 def queue(repo_label):
-    repo = g.repos[repo_label]
-    pull_states = sorted(g.states[repo_label].values())
+    if repo_label == 'all':
+        labels = list(g.repos.keys())
```

Comment:
This is what google told me the correct way to get a list of map keys is. If it's not, I can change it (homu is literally the only python I've ever touched, so if I do anything wonky, feel free to assume it's because I don't know what's going on).

I guess in this case `keys` is something iterable, and the extreme weak typing of all the variables means the types don't need to actually match up? Just that they both expose the same API?